article/sections/_7_latent_confounding.tex

```diff
@@ -0,0 +1,209 @@
+	prior on which variables are confounded. This can 
+	be in the form of building and testing candidate causal graphs that 
+	try to illustrate this confounding. 
+\end{itemize}
+
+
+We compare both the coefficient estimates on travel time and cost from 
+each of those three models to the true estimates used in the simulation. 
+We also compare the distribution of the recovered substitute confounder under each of 
+models 2 and 3 to the true confounder. The main findings of this exercise are the following:
+
+
+Using the true variables believed to be confounded (i.e. method 3 where only 
+travel time and cost are used to recover the confounder) leads to a better 
+recovery of the true confounder. Figure \ref{fig:qq-plot-method-2} and \ref{fig:qq-plot-method-3} show a QQ plot of the true 
+and recovered confounders under models 2 and 3 respectively. Looking at those 
+figures, we see that the distribution of the recovered substitute 
+confounder under method 3 is closer to that of the true confounder than method 
+2. This suggests that it may be better to run the deconfounder algorithm based 
+on a hypothesized causal graph, rather than just running it on all the 
+observed covariates. Please refer to Section \ref{sec:graph-construction} for how to build plausible causal graphs. 
+
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-2.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
+   \label{fig:qq-plot-method-2}
+\end{figure}
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{qq-plot-method-3.png}
+   \caption{QQ plot of true confounder against recovered confounder using all covariates}
+   \label{fig:qq-plot-method-3}
+\end{figure}
+
+% \begin{figure}
+% \centering
+% \begin{minipage}{.5\textwidth}
+%   \centering
+%   \includegraphics[width=.4\linewidth]{qq-plot-method-2.png}
+%   \captionof{figure}{QQ plot of true confounder against recovered confounder using all covariates}
+%   \label{fig:qq-plot-method-2}
+% \end{minipage}%
+% \begin{minipage}{.5\textwidth}
+%   \centering
+%   \includegraphics[width=.4\linewidth]{qq-plot-method-3.png}
+%   \captionof{figure}{QQ plot of true confounder against recovered confounder using only confounded covariates}
+%   \label{fig:qq-plot-method-3}
+% \end{minipage}
+% \end{figure}
+
+
+Additionally, and perhaps most importantly, the effectiveness of the 
+deconfounder algorithm is very sensitive to small errors and misfits in the 
+recovered confounder. Although method 3 returns a relatively good fit of the 
+true confounder (based on the QQ plot), the adjusted coefficients on travel 
+time and cost do not exhibit any reduction in the bias resulting from 
+ommitting the true confounder. Moreover, the coefficients on the recovered 
+confounder are highly insignificant. This raises questions about the 
+usefulness of the deconfounder algorithm in practice. Limitations notwithstanding, it is 
```

Comment:
Can you tell readers whether the standard errors on the regression coefficients were at least large? Are we wrong and confident or wrong and at least our model says we are uncertain?

Reply:
See edits in #177.